### ISSUE TITLE
Reference netstandard 2.0 package

### ISSUE THREAD

**Issue by fabiocav** [Azure/azure-webjobs-sdk-script#1368]
In order to load assemblies targeting .NET Standard 2.0, we need to make sure the runtime has a reference to the netstandard 2.0 NuGet package.

**Comment by fabiocav** [comment]
Blocked on package availability or decision to target net46

**Comment by fabiocav** [comment]
No longer needed.
